WordPressAuthenticator/GoogleSignIn/String+Random.swift

```diff
@@ -0,0 +1,39 @@
+extension String {
+
+    /// Returns a cryptographically secure string generated with characters from the given `Set<Character>` and with length
+    /// `length`.
+    ///
+    /// - Complexity: O(n) where n is the given `length`.
+    static func secureRandomString(using characters: Set<Character>, withLength length: Int) -> String? {
```

Comment:
I did a quick search and this function appears not being used anywhere, do we need it?

Reply:
That is correct.

I removed it in d83f7462 after your feedback. [_Kill your darlings_](https://www.goodreads.com/quotes/371112-kill-your-darlings-kill-your-darlings-even-when-it-breaks)